lib/travis/listener/app.rb

```diff
@@ -17,7 +17,15 @@ class App < Sinatra::Base
       # see https://github.com/github/github-services/blob/master/lib/services/travis.rb#L1-2
       # https://github.com/travis-ci/travis-api/blob/255640fd4f191f1de6951081f0c5848324210fb5/lib/travis/github/services/set_hook.rb#L8
       # https://github.com/travis-ci/travis-api/blob/255640fd4f191f1de6951081f0c5848324210fb5/lib/travis/api/v3/github.rb#L41
-      set :events, %w[push pull_request create delete repository installation installation_repositories]
+      set :events, %w[
+        push
+        pull_request
+        create
+        delete
+        repository
+        integration_installation
+        installation_repositories
+      ]
```

Comment:
I noticed that the :octocat: docs are incorrect; they claim that new installation events use the header `X-GitHub-Event: installation` when they actually use `X-GitHub-Event: integration_installation` ("Integrations" was the original name for "Apps")... and then the line was too long, so I reformatted it a bit